indico/modules/search/client/js/components/SearchApp.jsx

```diff
@@ -222,28 +223,49 @@ export default function SearchApp({category}) {
     trigger: 'once',
     camelize: true,
   });
+  let scope = {};
+  if (eventId !== null) {
+    scope = {event_id: eventId};
+  } else if (category !== null) {
+    scope = {category_id: category.id};
```

Comment:
Good idea to merge these into a scope argument in the hook above. But do we think we're going to need this scope check? As it doesn't seem wrong to send both at the same time. Or, even if one or both are undefined, I believe `useIndicoAxios` or the axios hook seems to correctly remove any undefined query params.

Reply:
Yeah, might work, but if for whatever reason both an event and a category gets passed (not that doing that would make any sense) you might end up sending both..